keys/keys_test.go

```diff
@@ -20,6 +20,7 @@ package keys
 import (
```

Comment:
I don't know why its mode bits modified to 755, I will change the mode bits back to 644.